### PR TITLE
Adds toolspeed

### PR DESCRIPTION
**Pull request by Firecage**
This PR adds a var named toolspeed and adds its usage to tools where it has a timer to do its function(except at tables...still couldn't get it to work). Still busy with this PR though.

Please inform me where else I can add this functionality to.

In short, you can make a tool construct/deconstruct things faster. Experimental welders, large crowbars, and nuke screwdrivers all does it twice as fast. Also works for badmining.

:cl:
rscadd: Hand tools now have variable operating speeds.
/:cl:

### CODE REVIEW

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -97,6 +97,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 	var/flags_cover = 0 //for flags such as GLASSESCOVERSEYES
 	var/heat = 0
 	var/sharpness = IS_BLUNT
+	var/toolspeed
```
Comment (RemieRichards) [inline code/game/objects/items.dm]: dividing by null? How about no.

Comment (Firecage) [inline code/game/objects/items.dm]: Kay.

**Comment by Iamgoofball** [comment]
Time for t2, t3, and t4 toolsets

**Comment by Firecage** [comment]
@Iamgoofball I swear on me mum....

**Comment by Jalleo** [comment]
@Iamgoofball no component based tools!

@Firecage table crafting stuff and wouldn't smaller crowbars be better for opening panels on machines?

**Comment by Firecage** [comment]
Jalleo. Well. Personally IIRL I have found that I work way faster with a large crowbar.

**Comment by bgobandit** [comment]
it's not the size of the tool...

**Comment by optimumtact** [comment]
rewrite construction to be datum recipes with this stuff all predone!

**Comment by Fox-McCloud** [comment]
I really like this idea.

**Comment by Robertha89** [comment]
Yes to component based tools

**Comment by Firecage** [comment]
"Still busy with this PR though."

**Comment by Jordie0608** [comment]
> 8 days ago

Make another.

**Comment by Firecage** [comment]
Jordie. Pretty sure there was still an issue or two I had to fix first...ahwell...

**Comment by optimumtact** [comment]
you can edit your pr to add [DNM] to the title, it almost guarantees your PR will not be merged

just a thought

**Comment by Firecage** [comment]
Eh, will do so on the next one.

**Pull request merged**